src/commands/schedule.ts

```diff
@@ -41,6 +41,12 @@ export const schedule: CommandWithInit = {
       return;
     }
 
+    const parsedDelete = Number(deleteTime);
```

Comment:
I prefer using `parseInt` here for parsing a string input to an integer value. Plus including the radix makes it explicit for the integer type we want: `parseInt(deleteTime, 10)`.